recipes/newrelic/infrastructure/suse.yml

```diff
@@ -141,9 +141,17 @@ install:
 
     install_infra:
       cmds:
+        - |
+          ARCH=$(uname -m)
```

Comment:
@brushknight are we trying to make sure the recipe does NOT run for ARM? There is an easier way to do this by adding a kernelArch to the installTargets, here is an example for Logs https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/infrastructure/logs/linux-logs.yml#L20